src/adapters/rhythmone.js

```diff
@@ -1,13 +1,12 @@
 var bidmanager = require('../bidmanager.js'),
   bidfactory = require('../bidfactory.js'),
-  utils = require('../utils.js'),
   CONSTANTS = require('../constants.json');
 
 import {ajax as ajax} from '../ajax';
 
-function track(debug, p1, p2, p3) {
+function track(debug) {
```

Comment:
We use google analytics to test this bidder on some of our own sites, and use this function for that purpose.  Having to add it and all its references back, only to remove it before a push, repeatedly, would be cumbersome.